js/navigation.jquery.js

```diff
@@ -304,6 +304,90 @@ class Navigation {
     // Handle in-page bookmarks.
     if (document.location.hash) this.hashChanged();
     $(window).on('hashchange', $.proxy(this.hashChanged, this));
+
+    // Accessibility
+    this.$container.on('keydown', (ev) => {
+      const $focus = $(document.activeElement);
+
+      const $li = $focus.parent();
+      const isEnterOrDown = ev.keyCode === 40 || ev.keyCode === 13;
+      const isEnterOrUp = ev.keyCode === 38 || ev.keyCode === 13;
+
+      // Is there a Bootstrap method to test this?
```

Comment:
Could listen for `shown.bs.dropdown` and `hidden.bs.dropdown` but your way is probably fine